app/src/rooms/views.py

```diff
@@ -16,11 +16,12 @@
 from conferences.models import UserPreferences, Zosia
 from rooms.forms import UploadFileForm
 from rooms.models import Room
-from rooms.serializers import room_to_dict, user_to_dict
+from rooms.serializers import room_to_dict
+from utils.functions import last_first_name_key
+from utils.views import csv_response
 
 
-# Cache hard (15mins)
-@cache_page(60 * 15)
+@cache_page(60 * 15)  # Cache hard (15mins)
```

Comment:
Why we are caching this?

Reply:
Wasn't this because we would've liked to simulate a "real-time" view of rooming?